snews_cs/snews_coinc.py

```diff
@@ -451,6 +458,13 @@ def hype_mode_publish(self):
 
     # ------------------------------------------------------------------------------------------------------------------
     def run_coincidence(self):
```

Comment:
We had a try and except method inside the `run_coincidence` but after exiting that we are using pymongo, and it crashed this morning. So I think it is safer to wrap the whole function into a try-except.